backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/HyperparametersFromConfigHandler.java

```diff
@@ -0,0 +1,86 @@
+package ai.verta.modeldb.experimentRun.subtypes;
+
+import static ai.verta.modeldb.entities.config.ConfigBlobEntity.HYPERPARAMETER;
+
+import ai.verta.common.KeyValue;
+import ai.verta.modeldb.common.futures.FutureJdbi;
+import ai.verta.modeldb.common.futures.InternalFuture;
+import ai.verta.modeldb.versioning.HyperparameterValuesConfigBlob;
+import com.google.protobuf.Value;
+import java.util.AbstractMap;
+import java.util.List;
+import java.util.concurrent.Executor;
+import java.util.stream.Collectors;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class HyperparametersFromConfigHandler extends KeyValueHandler {
+  private static Logger LOGGER = LogManager.getLogger(HyperparametersFromConfigHandler.class);
+  private final FutureJdbi jdbi;
+  private final Executor executor;
+
+  public HyperparametersFromConfigHandler(
+      Executor executor, FutureJdbi jdbi, String fieldType, String entityName) {
+    super(executor, jdbi, fieldType, entityName);
+    this.executor = executor;
+    this.jdbi = jdbi;
+  }
+
+  public InternalFuture<MapSubtypes<KeyValue>> getExperimentRunHyperparameterConfigBlobMap(
+      List<String> expRunIds, List<String> selfAllowedRepositoryIds) {
+    return jdbi.withHandle(
+            handle -> {
+              String queryStr =
+                  "SELECT distinct vme.experiment_run_id, hecb.name, hecb.value_type, hecb.int_value, hecb.float_value, hecb.string_value  FROM hyperparameter_element_config_blob hecb "
+                      + "INNER JOIN config_blob cb ON cb.hyperparameter_element_config_blob_hash = hecb.blob_hash "
+                      + "INNER JOIN versioning_modeldb_entity_mapping vme ON vme.blob_hash = cb.blob_hash "
+                      + "WHERE cb.hyperparameter_type = :hyperparameterType AND vme.experiment_run_id IN (<expRunIds>) ";
+
+              if (selfAllowedRepositoryIds != null && !selfAllowedRepositoryIds.isEmpty()) {
```

Comment:
This check looks weird. How do you differentiate between not having access to the repo and having access to all the repos (which is what this `if` assumes to be the case).

Reply:
What is the purpose of this comment?? `selfAllowedRepositoryIds` shows it self like it will comes from UAC  right. so `How do you differentiate between not having access to the repo and having access to all the repos` why you are confuse on it. this if added because if repo ids there then i have to add in query other wise do not add it. is simple as that. Why you are confuse here?

Reply:
How do you differentiate between not having access to the repo and having access to all the repos?

I am confused because I don't recall the answer to the question above. An empty list representing `all` and `none` in different cases is very confusing to me. Could you answer that question to remind me?

If you mean to have the behavior for all ids, would you mind using the new explicit field? https://github.com/VertaAI/protos-all/blob/master/protos/protos/private/uac/RoleService.proto#L135

Reply:
We have checked repository ids access here https://github.com/VertaAI/modeldb/pull/2219/files#diff-f554e4fd8e5f43fa49db0fa2ba8f6e7a5903f2259b01a1a1e411c21189a2ce78R837 and then I have passed accessible ids from here https://github.com/VertaAI/modeldb/pull/2219/files#diff-f554e4fd8e5f43fa49db0fa2ba8f6e7a5903f2259b01a1a1e411c21189a2ce78R855 so here i do not want to check again it has access or not. it has access that why those `selfAllowedRepositoryIds` ids are here. I think you do not have checked the flow OR if naming conventions are miss leading to you then you can suggestion me the name here?

Reply:
PR: https://github.com/VertaAI/modeldb/pull/2255